bin/reth/src/lib.rs

```diff
@@ -43,3 +44,38 @@ struct NetworkOpts {
     #[arg(long, value_delimiter = ',')]
     bootnodes: Option<Vec<NodeRecord>>,
 }
+
+/// Parameters for configuring the network more granularly via CLI
+#[derive(Debug, Args)]
+#[command(next_help_heading = "Rpc")]
+struct RpcServerOpts {
+    #[arg(long, default_value = "true")]
+    http: bool,
+
+    #[arg(long = "http.addr")]
+    http_addr: Option<String>,
+
+    #[arg(long = "http.port")]
+    http_port: Option<u128>,
+
+    #[arg(long = "http.api", value_delimiter = ',')]
+    http_api: Option<Vec<RethRpcModule>>,
+
+    #[arg(long, default_value = "true")]
+    ws: bool,
+
+    #[arg(long = "ws.addr")]
+    ws_addr: Option<String>,
+
+    #[arg(long = "ws.port")]
+    ws_port: Option<u128>,
+
+    #[arg(long = "ws.api", value_delimiter = ',')]
+    ws_api: Option<Vec<RethRpcModule>>,
+
+    #[arg(long, default_value = "true")]
+    ipcdisable: bool,
```

Comment:
this is a bit weird that this is opt-out?
does geth enable this by default?

Reply:
Not sure tbh, but Geth has the arg documented as 
```
--ipcdisable           Disable the IPC-RPC server
```

Reply:
so geth launches IPC by default @Rjected ?

let's keep it like this for now.